Task 2/Task 2.1/Task 2.1.2/RoundShapes/Ring.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+namespace Task_2._1._2.RoundShapes
+{
+    public class Ring : Figure
+    {
+        public Point Centre{ get => InnerCircle.Centre; }
+        public Circle OutherCircle { get; }
+        public Circle InnerCircle { get; }
+        public Ring(Point p, int innerR, int outherR) : base("Кольцо")
+        {
+            OutherCircle = new Circle(p, outherR);
+            InnerCircle = new Circle(p, innerR);
+        }
```

Comment:
No checks on correctness of parameters - incapsulation also means that you can instantiate only a correct object